webapp/channels/src/i18n/en.json

```
@@ -4929,7 +4929,6 @@
   "login.verified": " Email Verified",
   "manage_channel_groups_modal.search_placeholder": "Search groups",
   "manage_team_groups_modal.search_placeholder": "Search groups",
-  "mark_all_threads_as_read_modal.cancel": "Cancel",
   "mark_all_threads_as_read_modal.confirm": "Mark all as read",
   "mark_all_threads_as_read_modal.description": "This will clear the unread state and mention badges on all your threads. Are you sure?",
```

@pvev I think this needs to be updated
```suggestion
  "mark_all_threads_as_read_modal.description": "All your threads will be marked as read, with unread and mention badges cleared. Do you want to continue?",
```